articles/sso/oidc/single-page-apps-sso.md

```diff
@@ -0,0 +1,163 @@
+2. The **Create Client** window will open, allowing you to enter the name of your new Client. Choose **Single Page Web Applications** as the **Client Type**. When done, click on **Create** to proceed.
+3. Navigate to the [Auth0 Client Settings](${manage_url}/#/clients/${account.clientId}/settings) page. Add `http://localhost:3000` and `http://localhost:3000/callback.html` to the Allowed Callback URLs field of your [Auth0 Client Settings](${manage_url}/#/clients/${account.clientId}/settings).
+4. Scroll to the bottom of the [Settings](${manage_url}/#/clients/${account.clientId}/settings) page, where you'll find the *Advanced Settings* section. Under the *OAuth* tab, enable the **OIDC Conformant** Flag under the *OAuth* area of *Advanced Settings*.
+
+Now that you've configured your Auth0 Client, you can continue configuring your sample.
+
+1. Update the `auth0-variables.js` file included in the sample repository with your Auth0 Domain and the ID of the Auth0 Client you're using. These values can be found in your [Auth0 Client's Settings page](${manage_url}/#/clients/${account.clientId}/settings).
+2. Once you've made the configuration changes detailed in steps 2 and 3, start up a web server in the root of the repository at port `3000` .
+3. Browse to `http://localhost:3000` to view the client side of the sample.
+
+  ![Home page before logging in](/media/articles/sso/v2/spa/before-login.png)
+
+Please feel free to [download the sample](https://github.com/auth0-samples/oidc-sso-sample) and work through the examples on your local environment as you read this doc.
+
+## Silent Authentication
+
+Because client applications cannot query Auth0 directly to determine if users are logged in via SSO, the apps must redirect users to Auth0 for SSO authentication. However, because users find redirection disruptive, you should avoid doing so. One way of doing this is via **silent authentication**, which allows you to implement an authentication flow where Auth0 replies only with redirects (and never presents a login page to your users).
+
+### Configure Silent Authentication
+
+To bypass displaying the Lock screen when logging in a user (a process known as [*silent authentication*](#silent-authentication)), you must:
+
+* Enable the **Use Auth0 instead of the IdP to do Single Sign On** flag in the [Auth0 Client's settings page](${manage_url}/#/clients/${account.clientId}/settings);
+* Have a SSO cookie for the tenant's domain (in other words, the user has previously signed in and their saved cookie is still valid);
+* Pass the name of the user's Connection to Auth0 for authentication. You can do this by:
+  * Including it as a parameter when calling the `signin` function of the [auth0.js library](/libraries/auth0js);
+  * Passing the `connection` query string parameter when calling the [Authentication API's `/authorize` endpoint](/api/authentication#implicit-grant).
+
+### Initiate a Silent Authentication Request
```

Comment:
Can a comment about [renewAuth](https://github.com/auth0/auth0.js#api) be added for people using Auth0.js?